### PR TITLE
Set OnTimeBehavior in windowing strategy to FIRE_ALWAYS.

### PR DESCRIPTION
**Pull request by tvalentyn**
This fixes python side of https://issues.apache.org/jira/browse/BEAM-2961.

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by tvalentyn** [comment]
R: @lukecwik

**Comment by lukecwik** [comment]
CC: @robertwb 
It doesn't seem as though the Python SDK currently exposes the on time behavior in any meaningful way so it seems like we should always be using ALWAYS_FIRE. Note that this was the default before BEAM-2961 was fixed anyways.

File: sdks/python/apache_beam/transforms/core.py [hunk 1]
```diff
@@ -1548,6 +1548,7 @@ def to_runner_api(self, context):
         output_time=self.timestamp_combiner,
         # TODO(robertwb): Support EMIT_IF_NONEMPTY
         closing_behavior=beam_runner_api_pb2.ClosingBehavior.EMIT_ALWAYS,
+        OnTimeBehavior=beam_runner_api_pb2.OnTimeBehavior.FIRE_ALWAYS,
```
Comment (lukecwik) [inline sdks/python/apache_beam/transforms/core.py]: This is really weird that in the generated protos this one field is CamelCase and not snake_case.

**Comment by tvalentyn** [comment]
Thanks, @lukecwik !

**Pull request merged**